src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -317,4 +318,8 @@ public void flipCard() {
         updateFilteredCardList(new IsSameCardPredicate(currReview.getCurrCard()));
     }
 
+    @Override
+    public void tagCurrentCardInReview(Tag tag) {
+        currReview.tagCurrentCard(tag);
```

Comment:
Hmm I see. I think the issue is that I won't have the card index during the review session unlike edit card command. Can be discussed later!

Reply:
If I'm not wrong, in ModelManager, u don't need the card index because the filteredCard list should only have 1 card during review (the card under review).